app/listeners/organization.created/createAdmin.js

```diff
@@ -0,0 +1,22 @@
+var blueprint = require ('@onehilltech/blueprint')
+  , mongodb = require ('@onehilltech/blueprint-mongodb')
+  , ResourceController = mongodb.ResourceController
+  , users   = require ('../../../tests/fixtures/users')
+  ;
+
+var User = require ('../../models/User')
+  ;
+
+module.exports = exports = function (organization) {
+
+  adminData = users[2];
```

Comment:
I think a potential solution to this problem, is instead of listening for the organization.created event, in the postExecute function after creating the organization, we can create the admin account.  Then listen for the admin.created event and that is where we will send the email from.

So:
create organization request -> pre organization request (check for existing organization) -> create organization -> postExecute -> create admin -> listen for create.admin event -> send email.

The good thing about creating the admin in the postExecute is that it still has access to the request.  So if the admin email address is sent in the request as well, we can ignore it when creating the organization but will still have it in the request object in the postExecute.